packages/terra-table/tests/jest/ColumnHeader.test.jsx

```diff
@@ -178,8 +182,11 @@ describe('ColumnHeader', () => {
     );
 
     // Verify that column headers are present
-    const columnHeader = wrapper.find('.hidden');
-    expect(columnHeader).toHaveLength(0);
+    const hiddenSelector = wrapper.find('.hidden');
+    expect(hiddenSelector).toHaveLength(0);
+
+    const columnHeader = wrapper.find('.column-header-row');
+    expect(columnHeader.props()['data-row-id']).toBe('test-table-header-row');
```

Comment:
You probably don't need to add the check to this test.  The purpose of this test is just to ensure the column headers are not hidden.